app/code/community/Ebizmarts/MailChimp/Model/Api/Stores.php

```diff
@@ -25,8 +25,8 @@ class Ebizmarts_MailChimp_Model_Api_Stores
     public function createMailChimpStore($mailChimpStoreId, $listId = null, $scopeId, $scope)
     {
         $helper = $this->makeHelper();
-        $api = $helper->getApi($scopeId, $scope);
-        if ($api) {
+        try {
```

Comment:
Suggestion: move `$listId` checks up, before trying to get the API object. An `Ebizmarts_MailChimp_Helper_Data_ApiKeyException` could hide the more important exception here, that no lists are configured in MC.